rosidl_generator_cpp/cmake/rosidl_generator_cpp_generate_interfaces.cmake

```diff
@@ -74,20 +78,30 @@ rosidl_write_generator_arguments(
   TARGET_DEPENDENCIES ${target_dependencies}
 )
 
+get_filename_component(generator_module_path ${rosidl_generator_cpp_GENERATOR_FILES} DIRECTORY)
```

Comment:
`rosidl_generator_cpp_GENERATOR_FILES` can contain multiple files and should therefore not be interpreted as a single path.